src/Adapter/MSTest.TestAdapter/Execution/TestExecutionManager.cs

```diff
@@ -590,7 +590,7 @@ private async Task ExecuteTestsWithTestRunnerAsync(
             {
                 var result = new TestTools.UnitTesting.TestResult()
                 {
-                    Outcome = fixtureTestResult.Outcome.ToAdapterOutcome(),
+                    Outcome = fixtureTestResult.Outcome,
```

Comment:
`ToAdapterOutcome` is actually taking adapter outcome and converts it to TestFramework outcome, but anyways. Changed the type of `Outcome` property inside `FixtureTestResult` so that now no conversion is needed here.